common/services/prismic/projects.ts

```diff
@@ -5,12 +5,15 @@ import { parsePage } from './pages';
 import { Page } from '../../model/pages';
 import { PaginatedResults } from './types';
 import { IncomingMessage } from 'http';
+import { pagesFormatsFields } from '@weco/common/services/prismic/fetch-links';
```

Comment:
This is already in @weco.

Reply:
Good point. Does it matter? I guess using `@weco` everywhere makes everything more portable (and means you don't have to think about it).

Reply:
> everywhere makes everything more portable

Which it's not - the components come as a common package, not as discrete parts. I think this only works because of how babel is building things, and wouldn't work in a normal setting.

What this is saying is that a component in @weco has a dependency on an external package called @weco, which is a circular dep chain.

Reply:
ah, right-oh. Thanks, I'll update.